### ISSUE TITLE
Cannot play MKV on Calatyst

### ISSUE THREAD

**Issue by roman901** [monal-im/Monal#701]
Tried to open MKV file from chat, but cannot find a way _how_
<img width="1032" alt="Снимок экрана 2021-07-19 в 01 56 22" src="https://user-images.githubusercontent.com/2963008/126084653-eba3d47f-8916-4e94-9145-9673a15bc1cd.png">

**Comment by Echolon** [comment]
Hey @roman901 

MKV is a usually not supported natively by many vendors. Have you tried to play these files with for example VLC Player?

**Comment by roman901** [comment]
Hi @Echolon!
I’m shown that VLC player is installed, but there is no way to send it to it

**Comment by foss-** [comment]
Tested this briefly. Sending an mkv file results in an unplayable video. That aspect is expected, most other messengers like Signal also don't play mkv files.

**Currently**

When clicking the mkv file, macOS preview opens.

**Expected**

Current behavior is not great for mkv files since there is nothing useful in the quicklook preview.
Clicking on mkv files should open a file dialog, offering to store the file so it can then be opened  with a video player.

**Comment by tmolitor-stud-tu** [comment]
Yes, demoting mkv files from video to an arbitrary file transfer is possible.
